### DISCUSSION TITLE
POSTAL - Fix Mail-tester SpamAssassin error message

### DISCUSSION THREAD

**Discussion by rolix202** [postalserver/postal#2125] [Help with using Postal]
When I check my sending score using mail-tester, I got the following error messages

-0.683 | FROM_FMBLA_NEWDOM28 | From domain was registered in last 14-28 days

-0.626 | HK_RANDOM_ENVFROM | Envelope sender username looks random

-0.001 | SPF_HELO_NONE | SPF: HELO does not publish an SPF Record

How can I fix this, please?

**Comment by willpower232** [comment]
> From domain was registered in last 14-28 days

Presumably that will go away if you wait long enough.

> HK_RANDOM_ENVFROM

If this is a persistent problem for you, you could have a look at #1432 for making a small change to a value somewhere in your database. This is on my list of things to look at at some point.

> HELO does not publish an SPF Record

Have you added all the DNS records from https://docs.postalserver.io/install/dns-configuration ?
